assembl/fabfile.py

```diff
@@ -1528,7 +1765,8 @@ def webservers_start():
     if env.uses_nginx:
         # Nginx
         if exists('/etc/init.d/nginx'):
-            sudo('/etc/init.d/nginx start')
+            # Have to ensure that the env.user has visudo rights to call this
+            run('/etc/init.d/nginx start')
```

Comment:
even if the user has visudo right, you should prefix the command by `sudo` right? Like it's done for the reload.

Reply:
Well, from running reload/stop/start, it seems like it doesn't make much difference. If the user has visudo rights, they can call the command. I've made them all use sudo so it's consistent.